model-engine/model_engine_server/domain/use_cases/llm_model_endpoint_use_cases.py

```diff
@@ -370,10 +370,11 @@ def validate_checkpoint_path_uri(checkpoint_path: str) -> None:
 
 def get_checkpoint_path(model_name: str, checkpoint_path_override: Optional[str]) -> str:
     checkpoint_path = None
-    if SUPPORTED_MODELS_INFO[model_name].s3_repo:
-        checkpoint_path = get_models_s3_uri(SUPPORTED_MODELS_INFO[model_name].s3_repo, "")
+    models_info = SUPPORTED_MODELS_INFO.get(model_name, None)
```

Comment:
This LGTM but I'm worried there will be other errors if the model didn't have a preconfigured hardware profile. Looking forward to see what happens during testing!

Reply:
hardware is determined based off hugging face repo config so there's no need to preconfigure hardware profile other than special cases

Reply:
OK SG, will approve after seeing what the test plan and test results look like